apps/client/src/app/pages/portfolio/fire/fire-page.scss

```diff
@@ -1,3 +1,21 @@
 :host {
   display: block;
 }
+
+.swr-select {
+  background-color: transparent;
+  border: 0;
+
+  color: white;
```

Comment:
You cannot use `white` as a hardcoded color, in light mode it would be invisible.

Could you try to `inherit` the color? Otherwise you have to use

```
color: rgb(var(--dark-primary-text));
```
and
```
:host-context(.theme-dark) {
  color: rgb(var(--light-primary-text));
}
```